klippy/extras/bltouch.py

```diff
@@ -178,6 +187,44 @@ def home_start(self, print_time, sample_time, sample_count, rest_time):
                                            sample_count, rest_time)
     def get_position_endstop(self):
         return self.position_endstop
+    def set_output_mode(self, mode):
+        # If this is inadvertently/purposely issued for a
+        # BLTOUCH pre V3.0 and clones:
+        #   No reaction at all.
+        # BLTOUCH V3.0 and V3.1:
+        #   This will set the mode.
+        if mode == 'None':
+            return
```

Comment:
As above, this should be `if mode is None: return`

Reply:
Wilco